cmake/options.cmake

```diff
@@ -2,9 +2,10 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS "ON")
 set(CMAKE_MACOSX_RPATH 1)
 
 option(CMAKE_BUILD_TYPE "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel." "Release")
-option(VINECOPULIB_SHARED_LIB    "Pre-compiled version"              "OFF")
-option(WARNINGS_AS_ERRORS        "Compiler warnings as errors"       "OFF")
-option(OPT_ASAN                  "Use adress sanitizer (debug)"       "ON")
-option(BUILD_TESTING             "Build tests."                       "ON")
-option(CODE_COVERAGE             "Code coverage."                    "OFF")
-option(STRICT_COMPILER           "Stricter compiler warnings"        "OFF")
\ No newline at end of file
+option(VINECOPULIB_SHARED_LIB    "Pre-compiled version"              OFF)
+option(BUILD_SHARED_LIBS         "shared/static lib"                 ON)
```

Comment:
@jschueller I just noticed this one. Why is this needed? It doesn't seem to be used anywhere in the code base.

Reply:
its used by add_library if the latter does not specify SHARED or STATIC explicitely

Reply:
But we already have a "VINECOPULIB_SHARED_LIB",  should those two be separated ?

Reply:
(sorry again, we're cmake noobs)

Reply:
VINECOPULIB_SHARED_LIB enables the precompiled library, but BUILD_SHARED_LIBS decides if its shared or static

Reply:
Got it, thanks a lot!

Reply:
but maybe VINECOPULIB_SHARED_LIB should be renamed

Reply:
good idea, I opened an issue